server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -593,9 +601,15 @@ private static int compareName(String name, String repoName, SnapshotInfo info)
 
         private final List<SnapshotInfo> snapshotInfos;
 
-        SnapshotsInRepo(List<SnapshotInfo> snapshotInfos, boolean hasMore) {
+        private final int totalCount;
+
+        private final int remaining;
+
+        SnapshotsInRepo(List<SnapshotInfo> snapshotInfos, boolean hasMore, int totalCount, int remaining) {
             this.hasMore = hasMore;
             this.snapshotInfos = snapshotInfos;
+            this.totalCount = totalCount;
+            this.remaining = remaining;
```

Comment:
It think `hasMore` is redundant now that we have `remaining`?

Reply:
Right :) Will remove it